### PR TITLE
feat(ProductReviewsV2): added new review comment component - ReviewCommentV2

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Added `ReviewCommentV2` with some UI changes.

## Justify why they are needed

Part of [_Product Reviews v2_](https://www.figma.com/file/PUA7SuN75sWH4Q1QxDaeCJ/Product-Reviews?type=design&node-id=1447-1775&mode=design&t=Sri79oZDuQ3Tbqo4-4)

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3937?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #3930** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3930?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3937** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3937?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
      * **PR #3938** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3938?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3937?utm_source=stack-comment).

**Review by alebedev — APPROVED** [review]

File: apps/store/src/components/ProductReviews/ReviewCommentV2.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  const { t } = useTranslation('common')
+  const formatter = useFormatter()
+
+  return (
+    <div className={clsx(wrapper, className)}>
+      <Stars score={score} size="1rem" />
+
+      <Text className={reviewContent}>{content}</Text>
+
+      <footer className={footer}>
```
Comment (gustaveen) [inline apps/store/src/components/ProductReviews/ReviewCommentV2.tsx:25]: nit: maybe use a div instead if the footer isn't a child of a semantic element like article or section?

**Review by gustaveen — APPROVED** [review]
⭐️

File: apps/store/src/components/ProductReviews/ReviewCommentV2.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/components/ProductReviews/ReviewCommentV2.tsx:25]: Makes sense 👍🏻

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Feb 21, 5:27 AM EST**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3937).

**Pull request merged**